src/pages/home/slider/slider-home-page.style.js

```diff
@@ -101,21 +109,19 @@ export const useStyles = makeStyles(() => ({
     display: 'flex',
     flexDirection: 'column',
     justifyContent: 'space-between',
+    alignItems: 'flex-start',
     '@media (max-width: 768px)': {
-      minHeight: '5rem'
-    },
-    '@media (max-width: 320px)': {
-      minHeight: '5rem'
+      minHeight: '6rem'
```

Comment:
I think it's better to use px cause you used px before

Reply:
Agree